web/vital_records/models.py

```diff
@@ -239,9 +236,17 @@ def get_finished():
     @transition(field=status, source="initialized", target="started")
     def complete_start(self):
         self.started_at = timezone.now()
+        return Routes.app_route(Routes.request_type)
+
+    @transition(field=status, target="type_completed")
+    def complete_type(self, selected_type):
+        if selected_type == "birth":
+            return Routes.app_route(Routes.request_statement)
+        elif selected_type == "marriage":
+            return Routes.app_route(Routes.request_name)
         return Routes.app_route(Routes.request_statement)
```

Comment:
I wrote this code here just to see how one could pass in a form selection into the state method, and decide the next route based on the user's selection. But then, after I wrote this, I realized both Birth and Marriage flows go to the sworn statement next! Will we have a different sworn statement route/view for Birth and Marriage, or the same?